analytics/src/analytics/datasets/base.py

```diff
@@ -21,9 +21,14 @@ def from_dict(cls, data: list[dict]) -> Self:
         """Load the dataset from a list of python dictionaries representing records."""
         return cls(df=pd.DataFrame(data))
 
-    def to_csv(self, output_file: str) -> None:
+    def to_csv(
```

Comment:
Is this from running Black? Is Black not configured to run automatically?

Reply:
Black is auto-configured both in `make lint` and in the VSCode settings for the repo.

The reason this got bumped from a single line to multiple lines was because I added a few additional parameters:

**Before**
```python
def to_csv(self, output_file: str) -> None:
```

**After**
```python
def to_csv(
        self,
        output_file: str,
        *,  # force include_index to be passed as keyword instead of positional arg
        include_index: bool = False,
    ) -> None:
```